src/suites/cts/validation/error_scope.spec.ts

```diff
@@ -0,0 +1,106 @@
+export const description = `
+error scope validation tests.
+`;
+
+import { getGPU } from '../../../framework/gpu/implementation.js';
+import { Fixture, TestGroup } from '../../../framework/index.js';
+
+class F extends Fixture {
+  device: GPUDevice = undefined!;
+
+  async init(): Promise<void> {
+    super.init();
+    const gpu = getGPU();
+    const adapter = await gpu.requestAdapter();
+    this.device = await adapter.requestDevice();
+  }
+
+  createErrorBuffer(): void {
+    this.device.createBuffer({
+      size: 1024,
+      usage: 0xffff, // Invalid GPUBufferUsage
+    });
+  }
+
+  async expectUncapturedError(fn: Function): Promise<void> {
```

Comment:
What happens if we expectUncapturedError but we don't get one? Does this cause a test timeout?

The WPT runner has the concept of test timeouts - our standalone runner would just never complete the test.

Reply:
Good catch! I've added a timeout handler.

Reply:
:+1: